clients/drcachesim/common/options.h

```diff
@@ -58,6 +58,12 @@
 #define CACHE_TYPE_UNIFIED "unified"
 #define CACHE_PARENT_MEMORY "memory"
 
+#ifdef HAS_ZIP
+#    define DEFAULT_TRACE_COMPRESSION_TYPE "zip"
+#else
+#    define DEFAULT_TRACE_COMPRESSION_TYPE "gzip"
```

Comment:
If zip is not available lz4 could be the default if HAS_LZ4.  But gzip requires HAS_ZLIB right?  And I guess if none of the HAS_ is there the default would have to be "none".